### PR TITLE
General - Cleanup some unused/missing vars

### PR DESCRIPTION
**Pull request by PabstMirror**

### CODE REVIEW

**Review by johnb432 — APPROVED** [review]
Can't attest to the validity of the hemtt stuff, but the rest makes sense.

File: .hemtt/lints.toml [hunk 1]
```diff
@@ -8,3 +8,17 @@ options.ignore = [
 options.ignore = [
     "SLX_*", "ACE_*"
 ]
+
+[sqf.undefined]
+enabled = true
+options.check_oprhan_code = true
```
Comment (DartRuffian) [inline .hemtt/lints.toml:14]: Typo?

**Comment by johnb432** [comment]
@PabstMirror Did you run the tool with the typo or not?

**Comment by PabstMirror** [comment]
updated both to correct spelling - https://github.com/BrettMayson/HEMTT/pull/780/commits/20090060e4cfaf776f02702e7a197e2f4c21eed2

**Pull request merged**